src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -58,12 +59,23 @@ public async Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext
             DocumentCollectionInfo documentCollectionLocation;
             DocumentCollectionInfo leaseCollectionLocation;
             ChangeFeedProcessorOptions processorOptions = BuildProcessorOptions(attribute);
+            
             processorOptions.StartFromBeginning = attribute.StartFromBeginning;
+
             if (attribute.MaxItemsPerInvocation > 0)
             {
                 processorOptions.MaxItemCount = attribute.MaxItemsPerInvocation;
             }
 
+            if (DateTime.TryParse(attribute.StartFrom, out var startFromDateTime))
```

Comment:
Should it be only attempting parsing if the value is not null or whitespace? To avoid it altogether when its not really required

Reply:
updated